src/main/scala/com/databricks/spark/sql/perf/mllib/data/dataGeneration.scala

```diff
@@ -103,6 +103,30 @@ object DataGenerator {
 
     (sql.createDataFrame(trainPruned), sql.createDataFrame(finalTest))
   }
+
+  def generateRandString(
```

Comment:
We could then just rename `OverlappedStringGenerator` --> `StringGenerator` or `OverlappedStringGenerator` --> `RandStringGenerator`